### DISCUSSION TITLE
Cannot activate license because there is 1 database that uses documents compression

### DISCUSSION THREAD

**Discussion by wallaceturner** [ravendb/ravendb#18368] [General]
Am getting this message when i attempt to renew the community license on an *existing* RavenDb install 5.4.105 
```
Cannot activate license because there is 1 database that uses documents compression while the new license doesn't include the usage of documents compression
```

Can you please advise? I was not aware i was using documents compression (and do not require it)

**Comment by ppekrol** [comment]
Can you go over your databases and check which one has compression enabled? If you will disable it then, you should be able to activate your license.

**Reply by ayende** [reply]
Note that you may need to look at _revision_ compression as well

**Comment by wallaceturner** [comment]
which one of these settings? btw i note they are all the default

(we do not use revision functionality in either database)

![image](https://github.com/ravendb/ravendb/assets/2849980/7a9e35cb-0ba3-482e-9c6b-001ce562657e)

**Reply by ayende** [reply]
It's here

![image](https://github.com/ravendb/ravendb/assets/116915/b96c6563-853b-41c7-a5cb-79c068954b12)

**Comment by wallaceturner** [comment]
thanks working now - was set to true for one of the databases (even tho revisions were not present)
